Move GPG public key internal to role

msheiny opened the pull request:
We've been seeing lots of issues with the keyserver's bombing out and then the whole role fails. Lets start tracking the grsec pub GPG key internal to the role and drop the outbound network requirement.

msheiny committed:
Add Brad's GPG key directly into role defaults

msheiny committed:
Rely on internal GPG variable as opposed to keyserver

Keyserver's can be super flaky which causes the entire role to bomb out.
Moving the key internally to a variable seems like the right move going
forward.

conorsch commented on defaults/main.yml:
```diff
@@ -93,3 +92,97 @@ paxctld_configs_dist:
+  rDkzZpiYUYN6J/3gAGNw+Hj5ZqLlNmprLZBEsrK8FXhYb9+yh2YJNy2hD5AyGjMt
+  1bFx6/2R4TU=
+  =nWod
+  -----END PGP PUBLIC KEY BLOCK-----
```
Vars are great, I'd've stuck this in a file that we can clobber, then it's easier to import ad-hoc as well.

conorsch commented on tasks/main.yml:
```diff
@@ -1,6 +1,6 @@
 ---
 - name: Import Bradley Spengler GPG key (grsecurity maintainer) key
-  command: gpg --keyserver {{ paxctld_gpg_keyserver }} --recv-keys 44D1C0F82525FE49
+  shell: echo "{{paxctld_spengler_gpg|b64encode}}" | base64 -d | gpg --import
```
Pretty sure this'll work just as well:

```
- name: Import repository key to GnuPG.
  shell: gpg --import - <<< "{{ paxctld_spengler_key|escape }}"
  args:
    executable: /bin/bash # Necessary to support redirection.
```

conorsch commented on defaults/main.yml:
```diff
@@ -93,3 +92,97 @@ paxctld_configs_dist:
   /usr/lib/jvm/java-8-openjdk/jre/bin/java: m
   /lib/rc/bin/lsb2rcconf: E
+
+paxctld_spengler_gpg: |
```
Goes by `spender` in most grsec contexts, let's use that handle.

conorsch reviewed (changes requested):
a few cleanups, i'll handle in-place, then merge

conorsch committed:
Converts GPG pubkey to flat file

Easier to import, and simplifies the read-in-via-shell option for
importing the key. Set the lookup in the defaults file, so it's still
overrideable, as was the hardcoded vars approach.
